mindmeld/converter/dialogflow.py

```diff
@@ -25,6 +25,8 @@
 logger = logging.getLogger(__name__)
 package_dir = os.path.dirname(os.path.abspath(__file__))
 
+MIN_NUM_QUERIES = int(os.environ.get("MM_DF_CONVERTER_MIN_NUM_QUERIES", 5))
```

Comment:
I think what we actually want is an env variable that links the number of folds in the generic config file to the number of queries here

Reply:
Can you elaborate?

Reply:
resolved this issue